packages/pwa-kit-runtime/src/utils/ssr-server/utils.test.js

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
These tests are unrelated to the changes in the PR, but dropping the retry logic in the metrics code caused our branch coverage to drop. I decided to add more tests somewhere else, instead of dropping the coverage requirement.

The metrics code itself has 100% test and branch coverage.